app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
Yes 👍 

About `app/schemas`: Is that the correct place to put schemas? Also: In which gem should we package the schema? Should we add it to `diaspora_federation`? Or we can already create a separate gem for the schema `diaspora_federation-json_schema`, but release it together with everything else (like we already do for `-rails` and `-test`)?